src/main/resources/fabric-installer.launchwrapper.json

```diff
@@ -9,15 +9,15 @@
         "name": "net.minecraft:launchwrapper:1.12"
       },
       {
-        "name": "net.fabricmc:tiny-mappings-parser:0.1.1.8",
+        "name": "net.fabricmc:tiny-mappings-parser:0.2.0",
```

Comment:
This version should always match the build.gradle version 1:1, IMO.

Reply:
yeah, it's kinda annoying to change the version in 3 different places though

Reply:
We can use processResources gradle task's copy settings to replace in the future

Reply:
Yeah, these should really have the hash as well. For now leave it, and ensure they all match.